test/src/org/labkey/test/tests/targetedms/TargetedMSExperimentTest.java

```diff
@@ -54,7 +65,45 @@ public void testSteps()
 
         //small molecule
         importData(SKY_FILE_SMALLMOL_PEP, 3);
-        verifyImportedSmallMoleculeData();
+//        verifyImportedSmallMoleculeData();
```

Comment:
remove?

Reply:
I just re-enabled this. I had removed it to make it faster to test the updates.